### PR TITLE
Create requirements.txt

### PR DESCRIPTION
**Pull request by egc2846**

### CODE REVIEW

**Review by jpakey99 — APPROVED** [review]

File: backend/Makefile [hunk 1]
```diff
@@ -12,24 +12,39 @@
 OUT_DIR = dist/
```
Comment (Victoriam7) [inline backend/Makefile:12]: This comment is to hold merging until dependent pr is merged. That PR will also have a merge conflict once dependent PR is merged. Once OS changes are in feel free to close this comment

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+opencv-python==4.6.0.66
+argparse==1.4.0
+numpy==1.24.2
+scipy==1.10.1
+dataclasses
```
Comment (Victoriam7) [inline requirements.txt:5]: dataclasses is built in

**Review by Victoriam7 — CHANGES_REQUESTED** [review]

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+opencv-python==4.6.0.66
```
Comment (Victoriam7) [inline requirements.txt:1]: Could you also add easygui==0.98.3

File: backend/Makefile [hunk 4]
```diff
@@ -12,24 +12,39 @@
...
 DATA_DIR = data
```
Comment (Victoriam7) [inline backend/Makefile:13]: This file should be at the same level as the Makefile. backend/requirements.txt

**Review by Victoriam7 — CHANGES_REQUESTED** [review]

File: backend/Makefile [hunk 5]
```diff
@@ -12,24 +12,39 @@
...
+venv-init:
+	( \
+	 	python3 -m venv ./venv; \
+	 	. venv/bin/activate; \
+		pip install -r ../requirements.txt; \
+	)
+
+venv-clean:
+	rm -rf venv
+	find -iname "*.pyc" -delete
```
Comment (Victoriam7) [inline backend/Makefile:42]: why are we deleting .pyc files?

File: backend/Makefile [hunk 6]
```diff
@@ -12,24 +12,39 @@
...
 
 clean:
 	rm BeyondRGB.spec dist build out -rf
+	
+###########################VENV#######################################
+	
+venv-init:
+	( \
+	 	python3 -m venv ./venv; \
+	 	. venv/bin/activate; \
```
Comment (Victoriam7) [inline backend/Makefile:36]: This command fails when run in the make file. It works when run manually for some reason though.

**Review by Victoriam7 — CHANGES_REQUESTED** [review]

**Pull request merged**